packages/react-instantsearch/src/connectors/connectHits.js

```diff
@@ -17,17 +17,18 @@ export default createConnector({
   displayName: 'AlgoliaHits',
 
   getProvidedProps(props, searchState, searchResults) {
-    const index = getIndex(this.context);
     const results = getResults(searchResults, this.context);
     const hits = results ? results.hits : [];
 
-    return { hits };
+    return {
+      hits,
```

Comment:
prettier keeps it like this because you once wrote it in full, can you try writing it on one line and see what happens?